hamilton/function_modifiers/macros.py

```diff
@@ -1055,3 +1184,200 @@ def chain_transforms(
             )
             first_arg = raw_node.name
     return nodes, first_arg
+
+
+def apply_to(fn_: Union[Callable, str], **mutating_fn_kwargs: Union[SingleDependency, Any]):
+    """Creates an applicable placeholder with potential kwargs that will be applied to a node (or a subcomponent of a node).
+    See documentation for `mutate` to see how this is used. It de facto allows a postponed `step`.
+
+    :param fn: Function the applicable will be applied to
+    :param mutating_fn_kwargs: Kwargs (**kwargs) to pass to the mutator function. Must be validly called as f(**kwargs), and have a 1:1 mapping of kwargs to parameters.
+    :return: an applicable placeholder with the target function
+    """
+    return Applicable(fn=None, args=(), kwargs=mutating_fn_kwargs, target_fn=fn_, _resolvers=[])
```

Comment:
Not sure if the way I am using Applicable is ok or if I should subclass applicable and have a factory to create one child for `fn` and another child for `target_fn`

Reply:
So I think this is OK -- so long as it's documented as to why `Applicable` can have a null function. Seems to clean up a lot of code, no?

Re: subclass, there are two cases here and they're not strictly sub/supersets of each other. So subclass feels over-engineered (yeah, this was my original idea 😆). Should we get more patterns, we can figure it out, but I think this is likely about it.